test/autorest/generated/paginggroup/models.go

```diff
@@ -87,6 +91,11 @@ type ProductResult struct {
 
 // ProductResultResponse is the response envelope for operations that return a ProductResult type.
 type ProductResultResponse struct {
+	// PollUntilDone will poll the service endpoint until a terminal state is reached or an error is received
+	PollUntilDone func(ctx context.Context, frequency time.Duration) (*ProductResultResponse, error)
```

Comment:
So now this gets shared with synchronous APIs.  I have mixed feelings about this.  Probably worth discussing in scrum once Jeff is back.